posts/templates/posts/_post_create.html

```diff
@@ -13,16 +13,17 @@
     <br/>
 
     <div class="row">
-        {% if user.circles %}
-            <div class="form-group col-xs-12 col-md-4">
+        <div class="form-group">
+            <div class="col-xs-12 col-md-4">
                 <select class="form-control" name="circle">
-                    <option disabled selected="selected">Select a circle</option>
-                    {% for circle in user.circles|dictsort:'name' %}
-                        <option value="{{ circle.pk }}">{{ circle.name }}</option>
+                    <option value="-2"{% if post.circles_id == -2 %} selected{% endif %}>Only visible to me</option>
```

Comment:
I used the word "visible" to create the context in the first place. How would a new user know what the drop down is for when it only says "Me" and "Public", and there is no label next to it?